public/javascripts/SVValidate/src/panorama/Panorama.js

```diff
@@ -157,6 +158,22 @@ function Panorama (label) {
         return key in properties ? properties[key] : null;
     }
 
+    /**
+     * Gets the previous validated label from this Panorama.
+     * @returns     Last validated label from this mission.
+     */
+    function getLastLabel() {
+        return self.lastLabel;
+    }
+
+    /**
+     * Sets the previous label variable to a new label.
+     * @param newLastLabel Last validated label from this mission.
+     */
+    function setLastLabel(newLastLabel) {
+        self.lastLabel = {...newLastLabel};
```

Comment:
Is there a reason that this needs to be a shallow copy instead of just a direct reference to the label object?